polling_stations/apps/pollingstations/models.py

```diff
@@ -83,14 +83,15 @@ def get_polling_station_by_id(self, internal_council_id, council_id):
 
 
 class PollingStation(models.Model):
-    council             = models.ForeignKey(Council, null=True)
-    internal_council_id = models.CharField(blank=True, max_length=100)
-    postcode            = models.CharField(blank=True, null=True, max_length=100)
-    address             = models.TextField(blank=True, null=True)
-    location            = models.PointField(null=True, blank=True)
+    council = models.ForeignKey(Council, null=True, db_index=True)
+    internal_council_id = models.CharField(
+        blank=True, max_length=100, db_index=True)
```

Comment:
I did speed up queries on the polling station page, yes, but only by a little bit. You're right that it will never be a huge query, but I think for the time being the advantage of speeding up reads in anyway at all overweighs the disadvantage of slower imports. Can we keep it in for now and re-consider later?